src/js/letters/containers/AddressSection.jsx

```diff
@@ -13,34 +20,55 @@ export class AddressSection extends React.Component {
   }
```

Comment:
Not sure if it would make more sense for clarity to address in a separate tidy-up PR but since you're here...the state's getting initialized with `{ isEditAddressing: false }` - while I like this version better I think you meant `{ isEditingAddress: false }` 😛

Reply:
Ah yes! good catch, will fix this